src/main/java/duke/taskmanagement/TaskManager.java

```diff
@@ -0,0 +1,186 @@
+package duke.taskmanagement;
+
+import duke.customexceptions.EmptyDescriptionException;
+import duke.customexceptions.EmptyTimingDetailsException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class TaskManager {
+    private int taskUniqueID = 0;
+    private static ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
Another way to name this list is to use [plural form](https://se-education.org/guides/conventions/java/basic.html#naming).